mvp-spec/ad-server-implementation.md

```diff
@@ -0,0 +1,772 @@
+
+The Ad Server can then deserialize the cookie for having the Prebid SSO Data.
+
+Here is the structure of a Pseudonymous-Identifier:
+
+<!--partial-begin { "files": [ "identifier-table.md" ] } -->
+<!-- ⚠️ GENERATED CONTENT - DO NOT MODIFY DIRECTLY ⚠️ -->
+| Field   | Type          | Details                                            |
+|---------|---------------|----------------------------------------------------|
+| version | Number        | The version of PAF used.                                                                       |
+| type    | String        | The type of Pseudonymous-Identifier. For now, there is only one: "prebid_id".                                                    |
+| value   | String        | The Pseudonymous-Identifier value in UTF-8.                                                                                      |
+| source  | Source object | The Source contains all the data for identifying and trusting the Operator that generated the Pseudonymous-Identifier. <br /> <table><tr><th>Field</th><th>Type</th><th>Details</th></tr><tr><td>domain</td><td>String</td><td>The domain of the Operator.</td></tr><tr><td>timestamp</td><td>Integer</td><td>The timestamp of the signature.</td></tr><tr><td>signature</td><td>String</td><td>Encoded signature in UTF-8 of the Operator.</td></tr></table>|
+
+<!--partial-end-->
+
+Here is the structure of the Preferences:
+
+<!--partial-begin { "files": [ "preferences-table.md" ] } -->
+<!-- ⚠️ GENERATED CONTENT - DO NOT MODIFY DIRECTLY ⚠️ -->
+| Field   | Type                   | Details                                   |
+|---------|------------------------|-------------------------------------------|
+| version | Number                 | The Prebid SSO version used.     |
+| data    | Dictionary             | The keys are strings and represent the name of the preferences. <br /> The values represent the value of the preference. <br /> For now there is only one preference named "optin" and its value is a boolean.|
+| source  | Source object          | The source contains the data for identifying and trusting the CMP that signed lastly the Preferences.<br /> <table><tr><th>Field</th><th>Type</th><th>Details</th></tr><tr><td>domain</td><td>String</td><td>The domain of the CMP.</td></tr><tr><td>timestamp</td><td>Integer</td><td>The timestamp of the signature.</td></tr><tr><td>signature</td><td>String</td><td>Encoded signature in UTF-8 of the CMP.</td></tr></table>|
+
+<!--partial-end-->
+
+### Step 3: Generate the Seeds
+
+The Seed is the association of the Pseudonymous-Identifiers and the
+Preferences of the user for a given Addressable Content. The Ad Server must
+generate this association and sign it.
+Once generated, a Contracting Party - including the Ad Server- in possession of 
+a Seed can share it with another party via Transmissions (see the next steps). 
+A Seed is signed by the Ad Server for audit purposes. 
+
+Here is the composition of a Seed:
```

Comment:
It seems there is an inconsistency here in the definition of the seed as there is no Id or preferences fields fields in comparison with the definition from the dsp-api doc.